ironmon_tracker/data/PokemonData.lua

```diff
@@ -158,6 +160,11 @@ function PokemonData.initialize()
 					pokemonData.abilities = abilities
 				end
 			end
+			-- read in base friendship from Rom file for all mons - Tainted_wolf
+			local friendshipBase = PokemonData.readPokemonBaseFriendshipFromMemory(pokemonID)
+			if friendshipBase ~= nil then
+				pokemonData.friendshipBase = friendshipBase
+			end
```

Comment:
Just wanting to also add a comment here that this isn't checking if the base friendships have been randomized / modified.

Say for example the types get randomized but the base friendship remains vanilla, this codeblock as a whole will wind up reading *both* randomized types *and* base friendships from the ROM

Unsure if that's intentional or if we'd want that (since there is a defined default value later if the value is nil / not present in the pokemon data), just pointing it out

Reply:
Not individually, I lumped it in with the randomization because I don't know what i'm doing with this part of the code.

I think your comments have made me understand the pieces here fit together, so let me fix this and see if that fits with what you want.

Reply:
I got some work to do on this one... Give me a bit

Reply:
Okay, fixing a case typo was my issue. Also good to test with the right randomizer file that updates happiness.